etcdserver/config.go

```diff
@@ -181,9 +181,8 @@ func (c *ServerConfig) electionTimeout() time.Duration {
 }
 
 func (c *ServerConfig) peerDialTimeout() time.Duration {
-	// 1s for queue wait and system delay
-	// + one RTT, which is smaller than 1/5 election timeout
-	return time.Second + time.Duration(c.ElectionTicks)*time.Duration(c.TickMs)*time.Millisecond/5
+	// 1s for queue wait and election timeout
+	return time.Second + time.Duration(c.ElectionTicks)*time.Duration(c.TickMs)*time.Millisecond
```

Comment:
I am curious why  multiply `ElectionTicks` and `TickMs` `time.Duration(c.ElectionTicks)*time.Duration(c.TickMs)`?

Reply:
Generally looks good to me.

For readability, maybe go with:

```
time.Duration(c.ElectionTicks * c.TickMs)*time.Millisecond
```

This way we don't imply `c.ElectionTicks` is a duration (it's just count right?), until it's multiplied with the milliseconds per tick.

Reply:
> I am curious why multiply ElectionTicks and TickMs

to get the duration. ticks is number of ticks. tickMS is duration per key. you can read the code around tick to understand this better.

Reply:
@jpbetz fixed.